jc/parsers/xrandr.py

```diff
@@ -28,6 +27,16 @@
           "maximum_height":                    integer,
           "devices": [
             {
+              "props": {
+                [name: string]: {
+                  // Represents EdidModel
+                  "name": string,
+                  "product_id": string,
+                  "serial_number": string,
+                }
+                |
+                list[string]
```

Comment:
Not sure I understand this part of the schema.

Reply:
Oh, is this saying the `props` key will contain a key (arbitrarily named) with an object value OR a list of strings?

Reply:
Yeah, it can be either the specified EdidModel, which is formalized and predictable, or it can be any of the myriad props that can exist. The props category is so wide and unspecified that I'm just saying its values will be one or more line of strings.